src/qt/bitcoingui.cpp

```diff
@@ -1038,49 +1038,51 @@ void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate, double nVer
     progressBar->setToolTip(tooltip);
 }
 
-void BitcoinGUI::message(const QString &title, const QString &message, unsigned int style, bool *ret)
+void BitcoinGUI::message(const QString& title, QString message, unsigned int style, bool* ret)
 {
-    QString strTitle = tr("Bitcoin"); // default title
+    // Default title. On macOS, the window title is ignored (as required by the macOS Guidelines).
+    QString strTitle{PACKAGE_NAME};
```

Comment:
Why is this no longer translated, like the other `PACKAGE_NAME`s?

Reply:
I'm agree that in the `src/qt/` directory the translated `PACKAGE_NAME` is always used.
How should I fix it when this PR has been merged already? A new PR?

> Why is this no longer translated..?

The `PACKAGE_NAME` is a _brand/product name_ which should not be translated, in my strong opinion.
I've read rebranding discussion (#3203, #3400, #3408, ...).
Could you point me to a similar discussion regarding translation of the `PACKAGE_NAME`?

Also, please note the next lines of code
```
    <message>
        <source>Bitcoin Core</source>
        <translation>Bitcoin Core</translation>
    </message>
```
through almost(?) all files in the `src/qt/locale/` directory.

Maybe some national traditions force to translate foreign brand names? I don't know about such traditions.

Reply:
Yeah, there have been a lot of discussions in the past, also on IRC. We should at least be self-consistent.

Reply:
> We should at least be self-consistent.

Right.

Reply:
As #16291 is merged I'm going to resolve this conversation.